manager/README.md

```diff
@@ -198,7 +203,7 @@ MANAGER_QEMU_KERNEL_HASH=true \
 
 ### Verifying VM launch
 
-NB: To verify that the manager successfully launched the VM, you need to open two terminals on the same machine. In one terminal, you need to launch `go run main.go` (with the environment variables of choice) and in the other, you can run the verification commands.
+NB: To verify that the manager successfully launched the VM, you need to open three terminals on the same machine. In one terminal, you need to launch the Manager test server by executing go run ./test/manager-server/main.go (with the environment variables of choice), and in the second the manager by executing go run ./cmd/manager/main.go (with the environment variables of choice). Ensure that the Manager can connect to the Manager test server by setting the MANAGER_GRPC_PORT with the port value of the Manager test server. The Manager test server is listening on the default value of the MANAGER_GRPC_PORT. In the last one, you can run the verification commands.
```

Comment:
Put commands (`go run ...`) in verbatim, and maybe even on a separate line, for better readibility.

Reply:
Changed.